ironfish-rust/Cargo.toml

```diff
@@ -24,9 +24,12 @@ workspace = true
 workspace = true
 
 [features]
```

Comment:
nit: should this be called `proving` or `proof`? I am not sure `proofing` is correct here

Reply:
Renamed to `transaction-proofs`